src/sources/tw.json

```diff
@@ -12,5 +12,19 @@
         ],
         "active": false,
         "notes": "Updated API. Working on updating the adapter"
+    },
+    {
+        "url": "https://airtw.moenv.gov.tw/json/airlist/airlist_{i}_{dateStr}.json",
+        "adapter": "taiwan",
+        "name": "taiwan-moenv",
+        "city": "",
+        "country": "TW",
+        "description": "Taiwanese data from EPA",
```

Comment:
change this to not use a demonym for the data. Something like Data from the Tawain "moenv" (Ministry of Environment?)

Reply:
That is reasonable